tests/foreman/api/test_provisioning.py

```diff
@@ -534,20 +522,32 @@ def test_rhel_pxe_provisioning_fips_enabled(
     # Host is not blank anymore
     provisioning_host.blank = False
 
+    provisioning_host.key_filename = None
     # Wait for the host to be rebooted and SSH daemon to be started.
     provisioning_host.wait_for_connection()
 
+    # For RHEL9 since root login is disabled
```

Comment:
I see a lot of code repetition, and we'll probably need to use it in other tests too, so it's probably better to move this to https://github.com/SatelliteQE/robottelo/blob/master/robottelo/hosts.py#L1756 and only for RHEL9 Satellite.